src/components/menus/Items.ts

```diff
@@ -114,6 +115,29 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
         const options: any[] = [
             {
                 callback: (): void => {
+                    const category = this.gameStarter.itemsHolder.getItem(listing.item).category;
+                    if (category === "HM") {
+                        throw new Error("No onUse defined for HMs yet");
+                    }
+                    if (category === "TM") {
+                        throw new Error("No onUse defined for TMs yet");
+                    }
+                    const partyPokemon: IPokemon[] = this.gameStarter.itemsHolder.getItem("PokemonInParty");
+                    const chosenPokemon = partyPokemon[0];
+                    chosenPokemon.item = listing.item.split("");
```

Comment:
As a rule of thumb, if you're using `split("")` to convert into a name, that's a potential bug. See above.